pytest_remotedata/plugin.py

```diff
@@ -12,20 +12,23 @@ def pytest_addoption(parser):
     # The following means that if --remote-data is not specified, the default
     # is 'none', but if it is specified without arguments (--remote-data), it
     # defaults to '--remote-data=any'.
-    parser.addoption("--remote-data", nargs="?", const='any', default='none',
-                     help="run tests with online data")
+    parser.addoption(
+        "--remote-data", nargs="?", const='any', default='none',
+        help="run tests with online data")
 
 
 def pytest_configure(config):
     config.getini('markers').append(
-        'remote_data: Run tests that require data from remote servers')
+        'remote_data: Apply to tests that require data from remote servers')
+    config.getini('markers').append(
+        'internet_off: Apply to tests that should only run when network access is deactivated')
 
     # Monkeypatch to deny access to remote resources unless explicitly told
     # otherwise
-
     if config.getoption('remote_data') != 'any':
-        turn_off_internet(verbose=config.option.verbose,
-                          allow_astropy_data=config.getoption('remote_data') == 'astropy')
+        turn_off_internet(
+            verbose=config.option.verbose,
+            allow_astropy_data=config.getoption('remote_data') == 'astropy')
```

Comment:
I'm not sure the new formatting here is more readable - we usually aren't strict about 80 character line length, usually aiming for something like 100

Reply:
I go back and forth about this kind of thing all of the time. I'll leave this is as is but keep it in mind for the future.